community/kernel/src/main/java/org/neo4j/kernel/impl/proc/LoggingService.java

```diff
@@ -0,0 +1,26 @@
+package org.neo4j.kernel.impl.proc;
+
+/**
+ * LoggingService provides a logging facility for procedures. It should only be used
+ * within a procedure call
+ */
+public interface LoggingService
+{
+    boolean isDebugEnabled();
+
+    void error( String message );
+    void error( String message, Object... args );
```

Comment:
These guys need javadocs. For instance, there's no way to tell how the `message` should be formatted to interpolate and format the `args`.